src/GraphQL/Http/DocumentWriter.cs

```diff
@@ -11,48 +10,10 @@ public interface IDocumentWriter
         Task WriteAsync<T>(Stream stream, T value);
     }
 
-    public class DocumentWriter : IDocumentWriter
+    public static class DocumentWriterExtensions
```

Comment:
Please also change file name and namespace.

Reply:
Done